web/beacon-app/.gitignore

```diff
@@ -30,3 +30,11 @@ coverage
 public/tailwind.css
 
 #src/locales/*
+
+# cypress specific - we going to ignore all the files for now and only commit the ones we need
+cypress/fixtures/*
+cypress/downloads/*
+cypress/videos/*
+cypress/screenshots/*
```

Comment:
Some screenshots were committed below. Could they be removed so that they're now merged?